pykern/pkconst.py

```diff
@@ -33,27 +33,51 @@
 
 #: Copied from numconv, which copied from RFC1924
 BASE62_CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+#: PI
 pi = math.pi
+#: 2 * Pi
 TWO_PI = 2 * pi
+#: sqrt(TWO_PI)
 RT_TWO_PI = math.sqrt(TWO_PI)
+#: sqrt(2 / PI)
 RT_2_OVER_PI = math.sqrt(2 / pi)
+#: sqrt(2)
 RT_2 = math.sqrt(2.)
+#: speed of light
 C = scipy.constants.c
+#: C ^ 2
 C_SQ = C ** 2
+#: 1/ C
 C_INV  = 1. / C
+#: mu_0
 mu_0 = scipy.constants.mu_0
+#: epsilon_0
 epsilon_0 = scipy.constants.epsilon_0
+#: m_p
 m_p = scipy.constants.m_p
+#: m_e
 m_e = scipy.constants.m_e
+#: e
 e = scipy.constants.e
+#: 1 / (4 * PI * epsilon_0)
 MKS_factor = 1. / (4. * pi * epsilon_0)
+#: speed of light squared / e
 KG_to_EV = C_SQ / e
+#: 1.602e-12
 EV_to_ERG = 1.602e-12
+#: C * 10
 C_to_STATC = C * 10.
+#: e * C * 10
 E_CGS = e * C_to_STATC
-C_CGS = C * 100
-m_e_CGS = m_e * 1000.
-m_p_CGS = m_p * 1000.
+#: C * 100
+C_CGS = C * 100.
+#: m_e * 1000
+M_E_CGS = m_e * 1000.
+#: m_p * 1000
+M_P_CGS = m_p * 1000.
+#: m_e * C_SQ / e
 m_e_EV = m_e * KG_to_EV
```

Comment:
@robnagler @bruhwiler there seems to be a conflict with not using mixed case and with this line:

https://github.com/radiasoft/pykern/blob/f149c5d23696c3293e650147e5c70be3e1152a7f/pykern/pkconst.py#L79

 and this one:

https://github.com/radiasoft/pykern/blob/f149c5d23696c3293e650147e5c70be3e1152a7f/pykern/pkconst.py#L83

If we dont use mixed case then we have to rename one of these constants.

Reply:
The two variants are the same, except perhaps for a minus sign.
I recommend keeping M_E_EV = m_e * C_SQ / (-e) and deleting the other.